internal/engine/uid_map_manager_test.go

```diff
@@ -0,0 +1,124 @@
+package engine
+
+import (
+	"context"
+	"testing"
+	"time"
+
+	"github.com/windmilleng/tilt/internal/k8s/testyaml"
+
+	"k8s.io/apimachinery/pkg/watch"
+
+	"github.com/stretchr/testify/assert"
+
+	"github.com/windmilleng/tilt/internal/k8s"
+	"github.com/windmilleng/tilt/internal/model"
+	"github.com/windmilleng/tilt/internal/store"
+	"github.com/windmilleng/tilt/internal/testutils/output"
+)
+
+func _TestUIDMapManager(t *testing.T) {
```

Comment:
Does this test not pass?

Reply:
lol nope it does not

Reply:
Should we get it to pass before we merge it in? Have we looked in to why it doesn't pass?

Reply:
Ha okay, DM doesn't need this on master anytime soon so I will make this test be green